docs/nydusd.md

```diff
@@ -181,8 +181,8 @@ We are working on enabling cloud-hypervisor support for nydus.
       "type": "registry",
       "config": {
         ...
-        // Registry url scheme, https or http
-        "scheme": "http",
+        // Registry url scheme, leave empty to automatically detect, otherwise specify to https or http. The true represents https and false represents http
+        "scheme": true,
```

Comment:
This is an incompatible change, can we still keep it as a string and provides optional values `https` or `http` ?